android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java

```diff
@@ -573,6 +576,24 @@ private void showAboutDialog() {
                     .setTitle(appName)
                     .setCancelable(true)
                     .setPositiveButton(R.string.ok, null)
+                    .setNeutralButton(R.string.copy_settings,
+                            (dialog, which) -> {
+                                // Copy all settings to system clipboard.
+                                SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+                                StringBuffer allSettings = new StringBuffer("");
+                                allSettings.append(PrefUtil.getPrefsFromResource("Matching Engine Settings", getResources(), prefs, R.xml.pref_matching_engine));
+                                allSettings.append(PrefUtil.getPrefsFromResource("General Settings", getResources(), prefs, R.xml.pref_general));
+                                allSettings.append(PrefUtil.getPrefsFromResource("Speed Test Settings", getResources(), prefs, R.xml.pref_speed_test));
+                                allSettings.append(PrefUtil.getPrefsFromResource("Route Mode Settings", getResources(), prefs, R.xml.pref_route_mode));
+                                allSettings.append(PrefUtil.getPrefsFromResource("CV Network Settings", getResources(), prefs, R.xml.pref_network));
+                                allSettings.append(PrefUtil.getPrefsFromResource("CV UI Settings", getResources(), prefs, R.xml.pref_ui));
+                                allSettings.append(PrefUtil.getPrefsFromResource("CV Override Hostname Settings", getResources(), prefs, R.xml.pref_overrides));
+                                ClipboardManager clipboard = (ClipboardManager) getApplicationContext().getSystemService(Context.CLIPBOARD_SERVICE);
+                                ClipData clip = ClipData.newPlainText("simple text", allSettings);
+                                clipboard.setPrimaryClip(clip);

```

Comment:
"simple text" a bit too vage/arbitrary?

Would something like this work better as a label? "MobiledgeX Preferences"?